documentation/specs/BuildCheck/Codes.md

```diff
@@ -0,0 +1,43 @@
+# `BuildCheck` reports codes and their meaning
+
+Report codes are chosen to conform to suggested guidelines. Those guidelines are currently in revew: https://github.com/dotnet/msbuild/pull/10088
+
+| Exit&nbsp;Code | Reason |
+|:-----|----------|
+| 0 | Success |
+| [BC0101](#BC0101) | Shared output path. |
+| [BC0102](#BC0102) | Double writes. |
```

Comment:
This page seems to not have every code: https://github.com/dotnet/msbuild/blob/main/documentation/specs/proposed/BuildCheck/Codes.md